shared/src/main/scala/coop/rchain/pcasper/finalization/DagData.scala

```diff
@@ -0,0 +1,8 @@
+package coop.rchain.pcasper.finalization
+
+trait DagData[F[_], M, S] {
```

Comment:
It is good to describe the trait's responsibility.

Reply:
https://github.com/rchain/rchain/pull/3606